trade-finance-manager-api/src/graphql/helpers/percentageToNumber.helper.api-test.js

```diff
@@ -0,0 +1,33 @@
+const percentageToNumber = require('./percentageToNumber.helper');
+
+describe('percentageToNumber()', () => {
```

Comment:
Can you please cover a test case for percentage with decimals i.e. `80.2%`. Thanks.

Reply:
done

Reply:
done

Reply:
I can't see the changes in the PR